syncd/ComparisonLogic.cpp

```diff
@@ -2929,6 +2976,8 @@ void ComparisonLogic::createPreMatchMap(
     SWSS_LOG_NOTICE("preMatch map size: %zu, tmp oid obj: %zu",
             tmp.m_preMatchMap.size(),
             count);
+
+    cretePreMatchForAclEntries(cur, tmp, processed);
```

Comment:
Should this be before the log message above, so that the right prematch count gets printed?

Reply:
yes

Reply:
fixed